crates/cli/CHANGELOG.md

```
@@ -9,6 +9,37 @@ versioning through the workspace version in the root `Cargo.toml`.
 
 ### Changed
 
+- TUI: `R` on the selected message replies to it. It prefills `/reply ` in the composer (draft-protected like the `r`
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)